src/web/lib/braze/initialiseAppboy.ts

```diff
@@ -1,19 +1,33 @@
+import type appboy from '@braze/web-sdk-core';
```

Comment:
Note to ourselves to verify that this gives the behaviour we want!

Reply:
Discussed offline. Looking at the bundle size changes and visualisation I think this is fine.